src/main/java/com/github/javafaker/service/RandomService.java

```diff
@@ -50,4 +51,37 @@ public double nextDouble() {
     public Boolean nextBoolean() {
         return random.nextBoolean();
     }
+
+    /**
+     * Returns a random element from an array.
+     *
+     * @param array The array to take a random element fom.
+     * @param <E>   The type of the elements in the array.
+     * @return A randomly selected element from the array.
+     */
+    public <E> E nextElement(E[] array) {
+        return array[this.nextInt(array.length)];
+    }
+
+    /**
+     * Returns a random element from a list.
+     *
+     * @param list The list to take a random element fom.
+     * @param <E>  The type of the elements in the list.
+     * @return A randomly selected element from the list.
+     */
+    public <E> E nextElement(List<E> list) {
+        return list.get(this.nextInt(list.size()));
+    }
+
+    /**
+     * Returns a random enumeration value
+     *
+     * @param enumeration The enumeration to take a random value from
+     * @param <E>         The type of the enumeration
+     * @return A randomly selected enumeration value
+     */
+    public <E extends Enum<E>> E nextEnumValue(Class<E> enumeration) {
```

Comment:
just wanted to mention, that this duplicates https://github.com/DiUS/java-faker/blob/master/src/main/java/com/github/javafaker/Options.java#L15